spectests/structs_encoding.go

```diff
@@ -913,6 +922,9 @@ func (p *PendingAttestation) UnmarshalSSZ(buf []byte) error {
 
 	// Field (0) 'AggregationBits'
 	{
+		if o0 < 148 {
```

Comment:
sounds good

Reply:
To be clear, I interpreted "in `ReadOffset`" as "where we call `ReadOffset`", not actually *in* `ReadOffset`, as `ReadOffset` simply decodes the offset value from the byte slice with no bounds checking, and adding bounds checking to it would clutter its method signature (it would need to know about boundary offsets to perform checks).